dsl_lang/lib/ast.ml

```diff
@@ -0,0 +1,10 @@
+type clss = string
+
+type var = string
+
+type policy =
+| Class of clss
+| Fifo of policy list (* should change back to clss list? *)
```

Comment:
I think leaving it as `policy list` makes sense! It's more general than `clss list`, and someone can obviously create a list of `clss` using the `Class` constructor.